docs/introduction/index.rst

```diff
@@ -2,26 +2,73 @@
 Introduction to Flocker
 =======================
 
-Motivation for Building Flocker
-===============================
-Flocker lets you move your Docker containers and their data together between Linux hosts.
-This means that you can run your databases, queues and key-value stores in Docker and move them around as easily as the rest of your app.
-Even stateless apps depend on many stateful services and currently running these services in Docker containers in production is nearly impossible.
-Flocker aims to solve this problem by providing an orchestration framework that allows you to port both your stateful and stateless containers between environments.
+What is Flocker?
+================
 
-Docker allows for multiple isolated, reproducible application environments on a single node: "containers".
-Application state can be stored on a local disk in "volumes" attached to containers.
-And containers can talk to each other and the external world via specified ports.
+Flocker is an open-source Container Data Volume Manager for your Dockerized applications.
 
-But what happens if you have more than one node?
-How does application state work if you move containers around?
-Flocker solves this problem by moving your volumes to where your applications are.
+By providing tools for data migrations, Flocker gives ops teams the tools they need to run containerized stateful services like databases in production.
 
-The diagram below provides a high level representation of how Flocker addresses these questions.
+Unlike a Docker data volume which is tied to a single server, a Flocker data volume, called a dataset, is portable and can be used with any container, no matter where that container is running.
 
-.. image:: images/flocker-architecture-diagram.jpg
-   :alt: Containers run on physical nodes with Local Storage (ZFS).
-         Flocker's proxying layer allows you to communicate with containers by routing traffic to any node.
-         Filesystem state gets moved around with ZFS.
+Flocker manages Docker containers and data volumes together.
+When you use Flocker to manage your stateful microservice, your volumes will follow your containers when they move between different hosts in your cluster.
 
```

Comment:
And for this point you should specifically say "container", not "Docker", because if you're only interacting with dataset API it doesn't care about which container technology is using the datasets.